.circleci/config.yml

```diff
@@ -191,3 +191,7 @@ workflows:
                 - master
     jobs:
       - nightly_build
+
+notify:
+  webhooks:
+    - url: http://aleska-diamond.purposeindustries.co:8081/hubot/notification
```

Comment:
can you change it to: `https://hubot.purposeindustries.co:8081/hubot/notification`